features/FEATURE_BLE/targets/TARGET_Cypress/TARGET_CYW4343X/HCIDriver.cpp

```diff
@@ -24,6 +24,7 @@
 #include <stdbool.h>
 #include "hci_mbed_os_adaptation.h"
 #include "H4TransportDriver.h"
+#include "cycfg_pins.h"
```

Comment:
That file was used by [USI targets](https://github.com/ARMmbed/mbed-os/blob/ddc567d0d1fede0c18425689e48ae117a7a040f9/targets/targets.json#L2488-L2541) and is now inaccessible to these targets due to the `TARGET_Cypress` (these should be capital AFAIK). 

Please make sure the integration doesn't break existing targets.

Reply:
The TARGET_CYW4343X directory with HCI driver for targets inherited from USI_WM_BN_BM_22 was moved under TARGET_STM: https://github.com/ARMmbed/mbed-os/pull/9481/commits/22ed5e3d96f24c978bd4b7c446e3cb78ba1fecb0
I checked there is zero functional impact on MTB_USI_WM_BN_BM_22 and MTB_ADV_WISE_1530, thanks for noticing that before the CI.

features/FEATURE_BLE/targets now contains only vendor subdirectories. The TARGET_Cypress folder name is aligned with targets/TARGET_Cypress and features/mbedtls/targets/TARGET_Cypress